### PR TITLE
(complete) bootstrap & node-sass integration

### PR DESCRIPTION
**Pull request by shams-ali**
Please review bootstrap & node-sass integration

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,16 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "bootstrap": "4.3.1",
```
Comment (shams-ali) [inline package.json:6]: follow this tutorial for setting up bootstrap properly for react https://react-bootstrap.github.io/getting-started/introduction/

Comment (CyanBlueCode) [reply package.json:6]: Integrated react-bootstrap & imported bootstrap in App.js + CDN stylesheet in index.html

**Review by CyanBlueCode — APPROVED** [review]

**Review by shams-ali — COMMENTED** [review]
Please test the button

File: src/App.js [hunk 2]
```diff
@@ -2,6 +2,11 @@ import React from "react";
 import logo from "./logo.svg";
 import "./App.css";
 
+// bootstrap import
+import 'bootstrap/dist/css/bootstrap.min.css';
+// for buttons created in redux-thunk branch
+import { Button } from 'react-bootstrap';
```
Comment (shams-ali) [inline src/App.js:8]: delete this innnecessary line

**Pull request merged**